docs/docsite/rst/porting_guides/porting_guide_2.5.rst

```diff
@@ -199,12 +199,16 @@ Porting custom scripts
 
 No notable changes.
 
-Networking
-==========
+Network
```

Comment:
I think somewhere prominent in this section we should link to the network guide.